### PR TITLE
Create form

### PR DESCRIPTION
**Pull request by kabaskill**

### CODE REVIEW

File: components/Form.js [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    if (
+      !formData.name ||
+      !formData.image ||
+      !formData.categories ||
+      !formData.description ||
+      !formData.yearpublished ||
+      !formData.playtime ||
+      !formData.minAge ||
+      !formData.minPlayers ||
+      !formData.maxPlayers
```
Comment (moonwave99) [inline components/Form.js:38]: If you don't plan to display particularly custom error messages, prefer `required` on the fields

File: pages/api/games/index.js [hunk 2]
```diff
@@ -5,8 +5,17 @@ export default async function handler(request, response) {
...
-  } else {
+  }
+  if (request.method === "POST") {
+
+    try {
+      const formData = request.body;
+      console.log(formData);
 
-    return response.status(400).json({ error: "Bad Request" });
+      await Game.create(formData);
```
Comment (moonwave99) [inline pages/api/games/index.js:15]: `await Game.create(request.body)` is enough ^^

Also no log statements!

File: pages/profile/index.js [hunk 3]
```diff
@@ -1,18 +1,68 @@
...
+
+export default function ProfilePage({ toggleFavorite, isFavorite }) {
+  const [showModal, setShowModal] = useState(false);
+  const [content, setContent] = useState("form");
+  const { data: gameData, isLoading, error } = useSWR("/api/games");
+  let userCreatedGame = [];
+
+  if (gameData) {
+    userCreatedGame = gameData.filter((game) => game.userCreated === true);
+  }
```
Comment (moonwave99) [inline pages/profile/index.js:17]: More concise:

```js
const userCreatedGame = gameData
  ? gameData.filter(({ userCreated }) => userCreated)
  : [];
```

**Review by moonwave99 — CHANGES_REQUESTED** [review]

**Review by On-Fi — APPROVED** [review]
Looks good. 👍

**Pull request merged**